test/core/bitswap.spec.js

```diff
@@ -132,7 +141,7 @@ describe('bitswap', function () {
   })
 
   afterEach(function (done) {
-    this.timeout(80 * 1000)
+    this.timeout(160 * 1000)
```

Comment:
It still requires such big timeouts with the new js-ipfsd-ctl?

Reply:
- [ ] revert timeouts and also attempt to reduce them :)

Reply:
ping

Reply:
> Regarding the timeouts, we shouldn't spend time fiddling around with them but rather have a better solution for handling timeouts. My idea is to change from just a one-time check of values and timeout after X seconds, to a retrying-logic with backoff instead. Getting real tired of having random test failures because of timeouts...

Basically, spending our time fiddling around with fragile timeouts should not be our focus here or anywhere else...